bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/geometry_collector.cpp

```diff
@@ -155,119 +155,66 @@ mesh_data_t* GeometryCollector::startOrContinueMeshByFormat(uint32_t format)
 
 void GeometryCollector::addFace(
 	const std::vector<repo::lib::RepoVector3D64>& vertices)
+{
```

Comment:
do we still need this function? could we  just apply default parameters on the function it's calling with false?

Reply:
We could - I deliberately left it as an overload because not all combinations of the most general `addFace` are valid (e.g., uvs without normals). So the public methods correspond to valid uses, and the general one is private (with the intent that if someone tries to use it in another way in the future, they'll find its private and be forced to look at the implementation and so the comment that says there are limitations on how it's to be used!)

If you think the overload obfuscates it more than the alternative though, I can change it to use default parameters?

Reply:
@sebjf  I think it's fine as is, just looked a bit weird at first glance that it has to send false false to the other function.

But I think your argument of the separation and the latter function being a private function makes sense!